src/lib/multicall/executeMulticallWorker.ts

```diff
@@ -85,15 +91,24 @@ export async function executeMulticallWorker(
         `[executeMulticallWorker] Worker did not respond in time. Falling back to main thread. Job ID: ${id}`,
         request
       );
-      try {
-        const result = await executeMulticallMainThread(chainId, request);
+    } else {
+      emitMetricEvent({
+        event: "worker.multicall.error",
+        isError: true,
+        data: {
+          isInMainThread: false,
+          errorMessage: error.message,
```

Comment:
iirc in other places we also pass some other fields, let's pass them here as well 
<img width="620" alt="image" src="https://github.com/user-attachments/assets/83806c39-7734-49bd-85aa-4d9cd8e14a57">

Reply:
Added errorName and errorStack, but errorSource and errorStackHash are fields that the errorReporting module automatically adds to all unhandled errors caught at the window level. In this case, we have a handled error with a specific name, so I believe we can skip those fields